src/components/Locations.js

```diff
@@ -0,0 +1,70 @@
+import { Component } from "react";
+import Navbar from "./Navbar";
+
+
+
+class Locations extends Component{
+    constructor(){
+        super()
+        this.state = {
+            display: "Show locations",
+            allLocations: [],
+        };
+    };
+
+    fetchLocations = () => {
+        fetch("https://ghibliapi.herokuapp.com/locations")
+        .then((res) => res.json())
+        .then((data) => {
+            this.setState({
+                allLocations: data,
+
+            });
+        });
+    };
+
+    componentDidMount = () => {
+        this.fetchLocations();
+    };
+
+    handleButtonClick = () => {
+        this.setState({
+            display: "Hide locations"
+        })
+
+    } 
+
+    render(){
+
+        let locationsToDisplay = this.state.allLocations.map((location) => {
+            return(
+                <div className="location-display">
+                    <ul className="location-list">
+                        <li>
+                            <h3>Name: {location.name} </h3>
+                            <h3>Climate: {location.climate} </h3>
+                            <h3>Terrain: {location.terrain} </h3>
+                        </li>
+                    </ul>
+                </div>
```

Comment:
```suggestion
            return(
                        <li>
                            <h3>Name: {location.name} </h3>
                            <h3>Climate: {location.climate} </h3>
                            <h3>Terrain: {location.terrain} </h3>
                        </li>
```
Each location should be a single `li`. Once you've got all of your `li`s, they should all be contained inside of the same `ul`. This single `ul` should be part of the JSX in the return statement for the render() method. See below.